ChaosMod/Effects/db/Peds/PedsIntervention.cpp

```diff
@@ -0,0 +1,87 @@
+#include <stdafx.h>
+
+//Effect by ProfessorBiddle
+
+
+//future plans: make jesus spawn above and parachute to player, possibly jesus sniper in helicopter
+static void OnStart()
+{
+	Ped playerPed = PLAYER_PED_ID();
+	Vector3 playerPos = GET_ENTITY_COORDS(playerPed, true);
+	static constexpr Hash modelHash = -835930287;
+
+	Hash relationshipGroup;
+	ADD_RELATIONSHIP_GROUP("_DIVINE_JESUS", &relationshipGroup);
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, relationshipGroup, GET_HASH_KEY("PLAYER"));
+	SET_RELATIONSHIP_BETWEEN_GROUPS(0, GET_HASH_KEY("PLAYER"), relationshipGroup);
+	for (int i = 0; i < 4; i++)
+	{
+		//I am shit at programming if someone wants to do this in a more efficient way please do
+		//spawn 5 jesus
+		Ped ped0 = CreatePoolPed(4, modelHash, playerPos.x, playerPos.y, playerPos.z, GET_ENTITY_HEADING(playerPed));
+		Ped ped1 = CreatePoolPed(4, modelHash, playerPos.x + 5.f, playerPos.y, playerPos.z, GET_ENTITY_HEADING(playerPed));
+		Ped ped2 = CreatePoolPed(4, modelHash, playerPos.x - 5.f, playerPos.y, playerPos.z, GET_ENTITY_HEADING(playerPed));
+		Ped ped3 = CreatePoolPed(4, modelHash, playerPos.x, playerPos.y, playerPos.z + 5.f, GET_ENTITY_HEADING(playerPed));
+		Ped ped4 = CreatePoolPed(4, modelHash, playerPos.x, playerPos.y, playerPos.z - 5.f, GET_ENTITY_HEADING(playerPed));
+
+		auto playerGroup = GET_PLAYER_GROUP(PLAYER_ID());
+		
+		SET_PED_SUFFERS_CRITICAL_HITS(ped0, false);
+		SET_PED_RELATIONSHIP_GROUP_HASH(ped0, relationshipGroup);
+		SET_PED_HEARING_RANGE(ped0, 9999.f);
+		SET_PED_AS_GROUP_MEMBER(ped0, playerGroup);
+		SET_ENTITY_PROOFS(ped0, true, true, true, true, true, true, 1, true);
+		SET_PED_COMBAT_ATTRIBUTES(ped0, 5, true);
+		SET_PED_COMBAT_ATTRIBUTES(ped0, 46, true);
+		SET_PED_ACCURACY(ped0, 100);
+		SET_PED_FIRING_PATTERN(ped0, 0xC6EE6B4C);
+
+		SET_PED_SUFFERS_CRITICAL_HITS(ped1, false);
+		SET_PED_RELATIONSHIP_GROUP_HASH(ped1, relationshipGroup);
+		SET_PED_HEARING_RANGE(ped1, 9999.f);
+		SET_PED_AS_GROUP_MEMBER(ped1, playerGroup);
+		SET_ENTITY_PROOFS(ped0, true, true, true, true, true, true, 1, true);
```

Comment:
Should this not be `ped1`? Or am I mistaken about what `SET_ENTITY_PROOFS` is doing here? Just diving into the code + PRs for this project 🙂 
 
```suggestion
		SET_ENTITY_PROOFS(ped1, true, true, true, true, true, true, 1, true);
```

Also L53, L63, L73

Reply:
Yes it should be, I will fix that